tests/cli-v1.rs

```diff
@@ -99,7 +99,23 @@ fn list_toolchains() {
             &["rustup", "update", "beta-2015-01-01", "--no-self-update"],
         );
         expect_stdout_ok(config, &["rustup", "toolchain", "list"], "nightly");
+        expect_stdout_ok(config, &["rustup", "toolchain", "list", "-v"], "nightly");
+        expect_stdout_ok(
+            config,
+            &["rustup", "toolchain", "list", "--verbose"],
+            "nightly",
+        );
         expect_stdout_ok(config, &["rustup", "toolchain", "list"], "beta-2015-01-01");
+        expect_stdout_ok(
+            config,
+            &["rustup", "toolchain", "list", "-v"],
+            "beta-2015-01-01",
+        );
+        expect_stdout_ok(
+            config,
+            &["rustup", "toolchain", "list", "--verbose"],
+            "beta-2015-01-01",
+        );
```

Comment:
I don't really see anything in this suggesting that the verbose flag actually works, merely that it's not rejected by the command.  Could you augment the test to include at least the tab character so we can be somewhat confident that the verbosity is happening?  (Ditto that the tab is not present on a non-verbose listing)  Same applies to your cli-v2.rs changes.